### PR TITLE
adds prefrences to transhumanist

### PR DESCRIPTION
**Pull request by Skeleton-In-Disguise**
## About The Pull Request
You may remember this, that's because I accidentally deleted it before while trying to change things. Anyways! 
Adds drop-down selections and new options to transhumanist. also fixes a minor typo
Previously, you could choose your replaced limb by taking prosthetic limb, setting what you want changed, and then switching to transhumanist, since they used the same preference previously.

## Why It's Good For The Game

Transhumanist felt strange because it was hypothetically a voluntary operation, but the augmentation clinic just spun the wheel on what you got replaced. From a role-playing perspective, being unable to choose is uninteresting and confusing. Also it always says your limb being was being replaced with a robotic arm and that annoyed me. Now that you are able to select your replacement part, I've added two new options, the robotic voice box, good for a more prominent change then a limb that will be hidden for most of a round, and flashlight eyes, for when you are truly committed to being rushed directly to robotics seeing the bright future ahead of humanity!

## Changelog
:cl:
add: Transhumanist now allows you to select your augmentation
add: Transhumanist can now provide a robotic voice box, or flashlight eyes
spellcheck: Transhumanist's roundstart text has been re-written to not be wrong
/:cl:

### CODE REVIEW

**Comment by newfren** [comment]
flashlight eyes might be a noob trap?

File: code/datums/quirks/neutral_quirks/transhumanist.dm [hunk 1]
```diff
@@ -100,27 +101,59 @@
...
 
 	var/mob/living/carbon/human/human_holder = quirk_holder
-	var/obj/item/bodypart/new_part = new limb_type()
-
-	slot_string = "[new_part.plaintext_zone]"
-	old_limb = human_holder.return_and_replace_bodypart(new_part, special = TRUE)
+	var/obj/item/new_part = new part_type()
+	if(isbodypart(new_part))
+		var/obj/item/bodypart/new_bodypart = new_part
+		slot_string = [new_bodypart.plaintext_zone]
```
Comment (san7890) [inline code/datums/quirks/neutral_quirks/transhumanist.dm:117]:
```suggestion
		slot_string = new_bodypart.plaintext_zone
```

File: code/datums/quirks/neutral_quirks/transhumanist.dm [hunk 2]
```diff
@@ -100,27 +101,59 @@
...
+		old_part = human_holder.return_and_replace_bodypart(new_bodypart, special = TRUE)
+	else if(isorgan(new_part))
+		var/obj/item/organ/new_organ = new_part
+		old_part = human_holder.get_organ_slot(new_organ.slot)
+		if(new_organ.Insert(human_holder, special = TRUE))
+			old_part.moveToNullspace()
+			STOP_PROCESSING(SSobj, old_part)
+			slot_string = [new_organ.name]
```
Comment (san7890) [inline code/datums/quirks/neutral_quirks/transhumanist.dm:125]:
```suggestion
			slot_string = new_organ.name
```

**Review by san7890 — CHANGES_REQUESTED** [review]
this is why your code is failing to compile

**Comment by 00-Steven** [comment]
o7

**Comment by dopamiin0** [comment]
> flashlight eyes might be a noob trap?

ok but it's a really funny one and its not like its that hard to go get a fresh pair of eyes installed

**Comment by san7890** [comment]
i think "flashlights for eyes" is pretty descript enough, i was apprenhensive about it at first glance but it's not "flashlight eyes", the `for` does the heavy lifting there. also this will affect literally one round at most if people don't particularly like it, this is a game with a lot of zaniness (it would be sad if they got filtered by this but like... c'mon now there's far worse things) and one round of out of hundreds that any given player might partake in will not matter very much

**Pull request merged**